src/resolve-path.js

```diff
@@ -1,19 +1,23 @@
+import path from 'path';
 import url from 'url';
 
+const paths = {};
+
 const resolvePath = (request, urlBase) => {
   return new Promise((resolve, reject) => {
-    let { current } = request;
     const { previous } = request;
-    if (previous !== 'stdin' && current.substr(0, 5) !== 'jspm:') {
-      current = url.resolve(previous, current);
-    }
+    let { current } = request;
     if (current.substr(0, 5) === 'jspm:') {
       current = current.replace(/^jspm:/, '') + '.scss';
       System.normalize(current)
-        .then(path => resolve(path.replace(/\.js$|\.ts$/, '')))
+        .then(p => resolve(p.replace(/\.js$|\.ts$/, '')))
```

Comment:
I had to rename it because I imported `path` above so I could call `path.dirname`.